src/icrar/leap-accelerate/algorithm/cpu/PhaseRotate.cc

```diff
@@ -104,13 +103,23 @@ namespace cpu
             {
                 double shiftRad = shiftFactor / metadata.m_constants.channel_wavelength[channel];
 
-                data(channel, baseline) *= std::exp(std::complex<double>(0.0, 1.0) * std::complex<double>(shiftRad, 0.0));
+                for(int polarization = 0; polarization < integration_data.dimension(2); ++polarization)
+                {
+                    integration_data(channel, baseline, polarization) *= std::exp(std::complex<double>(0.0, 1.0) * std::complex<double>(shiftRad, 0.0));
+                }
```

Comment:
I suspect it should be possible to use `shiftRad * 1.0i`

Reply:
That's also true, and even better, but I didn't mention it because I thought it came with 17 -- but it's 14, so it should be alright. You'd need to add a `using namespace std::complex_literals` at the beginning of your function though.

Reply:
Bugger, `i` is a [GNU extension](https://gcc.gnu.org/onlinedocs/gcc/Complex.html#Complex) so we'd have to explicitly use `-std=c++14` instead of `-std=gnu++14` (probably what cmake generates) in the compiler options. A battle for the future...